src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -163,6 +163,21 @@
   // * Menu toggle small
   --#{$menu-toggle}--m-small--PaddingTop: var(--pf-t--global--spacer--xs);
   --#{$menu-toggle}--m-small--PaddingBottom: var(--pf-t--global--spacer--xs);
+
+  // Status icon
+  --#{$menu-toggle}__status-icon--MarginInlineEnd: var(--pf-t--global--spacer--md);
+
+  // Success
+  --#{$menu-toggle}--m-success--after--BorderBottomColor: var(--pf-t--global--border--color--status--success--default);
```

Same would apply to warning and danger vars + where they're used:


```suggestion
  --#{$menu-toggle}--m-success--BorderColor: var(--pf-t--global--border--color--status--success--default);
```

---

d'oh! copy/pasta fail 🤦‍♂️